src/utils/url.ts

```diff
@@ -37,5 +37,10 @@ export const isRemoveJourney = (req: Request): boolean => {
     }
   }
 
+  // UAR-1352 Only required for remove feature flag - can be removed with it
+  if (!req.query[config.JOURNEY_QUERY_PARAM] && req.url.includes("remove")) {
```

Comment:
I think looking for a string 'remove' in the URL is too risky - it could be present if on the update or registration journeys (e.g. if there was a page to remove a trust/BO etc).

The idea was to explore storing the 'isRemove=true' field in the session on the very first screen of the remove journey and then check for that in this method. You might want to try that.

Reply:
Would this mean getting rid of the query parameter altogether?

Reply:
Potentially. I think in the first instance it would be good to just see if the approach would work and how much effort it is (perhaps ignoring unit-test changes to start with). Maybe on a new branch.